addons/test_mail/tests/test_mail_followers.py

```diff
@@ -213,21 +213,6 @@ def test_followers_inverse_message_partner_access_rights(self):
         with self.assertRaises(AccessError):
             test_record.message_partner_ids -= partner0
 
-    @users('employee')
-    def test_followers_private_address(self):
```

Comment:
Could be interesting to migrate the test to: "use api to subscribe IDs the user can't read" (for both high-level and low-level api). Idea would be: message_subscribe should raise, _message_subscribe probably not. As for other tests actually, we could patch the check access on partner just to see how it behaves.